python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -298,14 +298,16 @@ def remove(self):
         if self.lfn == "":
             raise GangaException('Can\'t remove a  file from DIRAC SE without an LFN.')
         logger.info('Removing file %s' % self.lfn)
-        stdout = execute('removeFile("%s")' % self.lfn)
-        if isinstance(stdout, dict) and stdout.get('OK', False) and self.lfn in stdout.get('Value', {'Successful': {}})['Successful']:
-            self.lfn = ""
-            self.locations = []
-            self.guid = ''
-            return
-        logger.error("Error in removing file '%s' : %s" % (self.lfn, stdout))
-        return stdout
+        try:
+            stdout = execute('removeFile("%s")' % self.lfn)
+        except GangaDiracError as err:
+            logger.error("Error in removing file '%s' : %s" % (self.lfn, err))
+            raise
+
+        self.lfn = ""
+        self.locations = []
+        self.guid = ''
+        return
```

Comment:
Actually I think this should be True as we expressed an interest that the remove command as well should return True/False depending on whether it succeeded or not.